_pages/tutorials/webinars.md

```diff
@@ -22,11 +22,15 @@ Whether you're looking to supercharge your application with readily-usable Machi
 
 Algorithmia's AI Layer and Marketplace make it easy to get Machine Learning up and running in your app. Our Webinar will show you how.
 
-<a href="https://register.gotowebinar.com/rt/6993672803865799683" class="btn btn-default btn-primary">Register Now</a>
+<a href="https://register.gotowebinar.com/rt/6993672803865799683">
+  <button class="syn-btn contained theme-primary">Register Now</button>
+</a>
 
 
 ### Deploying Machine Learning The Right Way With The AI Layer
 
 Most Machine Learning doesn't make it past Powerpoint. We'll show you how our AI Layer can push your models into production and free up your team to focus on what matters.
 
-<a href="https://register.gotowebinar.com/rt/897829706382697987" class="btn btn-default btn-primary">Register Now</a>
+<a href="https://register.gotowebinar.com/rt/897829706382697987">
+  <button class="syn-btn contained theme-primary">Register Now</button>
+</a>
```

Comment:
This could be a good opportunity to create the styles necessary for links to render similar to buttons.

Using this markup:
```
<a
    href="https://register.gotowebinar.com/rt/897829706382697987"
    class="syn-btn contained theme-primary"
>
    Register Now
</a>
```
We could create a new class `.syn-link-button` that has the following styles:
```
    display: block;
    width: fit-content;
```
It renders exactly the same and lets us remove the `<button>` element entirely. I'm sure it'd be a PITA to update all possible Synaptic button styles, but this would be an excellent start.

Reply:
Updated: actually just had to add `display: inline-block;` to get the link-buttons to work.